web/packages/teleport/src/Users/Users.tsx

```diff
@@ -49,16 +49,31 @@ export function Users(props: State) {
     onUpdate,
     onDelete,
     onReset,
+    onStartInviteCollaborators,
+    onInviteCollaboratorsClose,
+    inviteCollaboratorsOpen,
```

Comment:
We can destructure `inviteCollaborators` here to follow the other props.

Reply:
Sure, I'd done it this way since it made the type checker angry, turns out intrinsic elements just need to be capitalized? :facepalm:  Fixed now at least.